automatic/webstorm/webstorm.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>webstorm</id>
+    <version>2017.1.3.20170526</version>
+    <title>JetBrains WebStorm</title>
+    <authors>JetBrains</authors>
+    <licenseUrl>http://www.jetbrains.com/webstorm/buy/buy.jsp</licenseUrl>
+    <projectUrl>https://www.jetbrains.com/webstorm/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/8edfc93894e9bdd11deb36d1cbbfe8a47a7b013a/icons/webstorm.png</iconUrl>
+    <docsUrl>https://www.jetbrains.com/help/webstorm</docsUrl>
+    <summary>WebStorm — The smartest JavaScript IDE</summary>
+    <releaseNotes>https://www.jetbrains.com/webstorm/whatsnew/</releaseNotes>
+    <tags>webstorm admin jetbrains trial 30days developer IDE HTML CSS JavaScript</tags>
+    <bugTrackerUrl>https://youtrack.jetbrains.com/issues/WEB</bugTrackerUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+WebStorm — The smartest JavaScript IDE. Lightweight yet powerful IDE,
```

Comment:
Per [1.2.3](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description) you should move this into `README.md` and have AU load it during the build.